enhancements/olm-toggle-copied-csvs.md

```diff
@@ -75,23 +75,19 @@ That is, all CSVs installed by a given `Subscription` share an `Operator` resour
 
 Orphaned CSVs are CSVs that were installed via a `Subscription` that no longer exists. Since the lifetimes -- i.e. the period of existence on a cluster --  of `Operator` resources aren't directly tied to the lifetimes of the resources that triggered their generation, the `Operator` resource(s) associated with Orphaned CSVs will continue to exist _after_ a `Subscription` is deleted. If that `Operator` resource is deleted explicitly, OLM will generate another as per the rules [outlined above](#operator-cardinality).
 
-#### Copied RBAC 
-
-In addition to operator discoverability, Copied CSVs also serve as convienient vehicles for the garbage collection of Copied RBAC. These are resources that are copied along with the CSV to provide the operator access to a namespace. To support the loss of Copied CSVs, the RBAC copying logic in OLM will be extended to write additional owner references from copied resources to the related Operator resource. 
-
 #### The Toggle
 
 A feature toggle will be added to control OLM's use of Copied CSVs. When enabled (toggled on), OLM will delete all existing Copied CSVs and will not generate any more. When disabled (toggled off), OLM will generate Copied CSVs (much like it does today).
 
-When toggled (on then off), OLM will recreate any missing Copied CSVs and reassociate the owner references of related copied resources with them (i.e. the inverse operation of the one described in the [Copied RBAC](#copied-rbac) section).
+When toggled (on then off), OLM will recreate any missing Copied CSVs.
 
 The toggle will be disabled by default.
 
 The toggle can be set from a command-line option on the olm-operator binary.
 
 #### Config CRD w/ Toggle
 
-A novel cluster-scoped `OLMConfig` CRD will be added to OLM. The resource type it defines will allow users to apply a limited set of configurations to the OLM instance on their cluster by creating or modifying a singlton (of that type) on the cluster.
+A novel cluster-scoped `OLMConfig` CRD will be added to OLM. The resource type it defines will allow users to apply a limited set of configurations to the OLM instance on their cluster by creating or modifying a singleton (of that type) on the cluster.
```

Comment:
Can we also update what version of the OLMConfig API will be introduced?

Reply:
I believe that OpenShift requires it to be introduced at v1.

Reply:
I'm definitely concerned with introducing a new API at a stable version, especially around any API with a high surface (e.g. one that drives configuration for the rest of the stack), but I can get behind those potential pain points as long as we're careful with how we expand the surface.